common/lib/common-utils/src/bufferBrowser.ts

```diff
@@ -93,12 +93,13 @@ export class IsoBuffer extends Uint8Array {
         // Capture any typed arrays, including Uint8Array (and thus - IsoBuffer!)
         } else if (value !== null && typeof value === "object" && isArrayBuffer(value.buffer)) {
             // Support currently for full array, no view ports! (though it can be added in future)
-            assert(value.byteOffset === 0, 0x000 /* "nonzero isobuffer byte offset" */);
-            assert(value.byteLength === value.buffer.byteLength, 0x001 /* "unexpected isobuffer byte length" */);
+            assert(value.byteOffset === 0, 0x0_00 /* "nonzero isobuffer byte offset" */);
+            assert(value.byteLength === value.buffer.byteLength, 0x0_01 /* "unexpected isobuffer byte length" */);
```

Comment:
[unicorn/numeric-separators-style](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v40.0.0/docs/rules/numeric-separators-style.md)

Reply:
I like this rule for readability, but I think it might interfere with the automatic assert message stuff that @anthony-murphy added awhile back.

Reply:
Woah, freaky.  I had no idea you could put `_` in the middle of a number.  I'm not sure I like it...

As for assert tags, yeah this is a no-go for me.  It hinders readability in this case.  If consensus moves in favor of this rule, then at least let's update the config for hex to be every 4 digits or something?